backend/routes/userRoutes.js

```diff
@@ -0,0 +1,58 @@
+import { Router } from 'express'
+import { db } from '../lib/db'
+import { users_db_name } from '../Utilities/API_utilities'
+
+
+const usersRouter = Router()
+
+
+usersRouter.post('/api/user', (req, res) => {
```

Comment:
Do we want only admins to be able to create new users?

Reply:
Yes but need to lock down routes in another PR